runtime/gc_glue_java/configure_includes/configure_aix_ppc.mk

```diff
@@ -77,6 +77,10 @@ CONFIGURE_ARGS += 'OMR_HOST_OS=aix'
 CONFIGURE_ARGS += 'OMR_HOST_ARCH=ppc'
 CONFIGURE_ARGS += 'OMR_TARGET_DATASIZE=$(TEMP_TARGET_DATASIZE)'
 CONFIGURE_ARGS += 'OMR_TOOLCHAIN=xlc'
-
-CONFIGURE_ARGS+= 'GLOBAL_CFLAGS=-qstackprotect'
-CONFIGURE_ARGS+= 'GLOBAL_CXXFLAGS=-qstackprotect'
+ifeq ($(OMR_ENV_OPENXL),1)
+	CONFIGURE_ARGS+= 'GLOBAL_CFLAGS=-fstack-protector'
+	CONFIGURE_ARGS+= 'GLOBAL_CXXFLAGS=-fstack-protector'
+else
+	CONFIGURE_ARGS+= 'GLOBAL_CFLAGS=-qstackprotect'
+	CONFIGURE_ARGS+= 'GLOBAL_CXXFLAGS=-qstackprotect'
+endif
```

Comment:
Lines in makefiles should not begin with a tab except within a script; please insert a space before each `+=` (see line 79):
```
ifeq ($(OMR_ENV_OPENXL),1)
  CONFIGURE_ARGS += 'GLOBAL_CFLAGS=-fstack-protector'
  CONFIGURE_ARGS += 'GLOBAL_CXXFLAGS=-fstack-protector'
else
  CONFIGURE_ARGS += 'GLOBAL_CFLAGS=-qstackprotect'
  CONFIGURE_ARGS += 'GLOBAL_CXXFLAGS=-qstackprotect'
endif
```

Reply:
I have added the space before +=. I initially used the tab as the previous ifeq condition checkings also used tabs instead of space. Instead of tab now I added 4 spaces